docs/releasenotes.md

```diff
@@ -23,9 +35,9 @@ OpenEBS Release Version 0.5.1
 -   Not recommended for mission critical workloads
 -   Not recommended for performance sensitive workloads. Efforts are ongoing to improve performance.
 
-### Installation
+### Enhancements to Installation
 
-Execute the following command to install OpenEBS using kubectl.
+Enhancements to helm charts were added in this release. Execute the following command to install OpenEBS using kubectl.
```

Comment:
this seems out of place.

Reply:
Removed the line "Enhancements to helm charts were added in this release."